src/utils/vc/crypto/common/DockCryptoSignature.js

```diff
@@ -135,7 +133,18 @@ export default withExtendedStaticProperties(
         // }
       }
 
+      // Else, schema object exists but no ID means the SDK is signalling for the suite to generate a schema
+      if (!credSchema && document.credentialSchema) {
+        credSchema = new CredentialSchema(CredentialSchema.essential());
+      }
+
+      // Else, no schema was found so just use the essentials and v0.0.1 schema version
+      // NOTE: version is important here and MUST be 0.0.1 otherwise it will invalidate BBS+ credentials
```

Comment:
can we have a test that enforces the schema version?

could there ever be a need to have different version? If so, what does that look like?

Reply:
we already have a test to make sure verifying that works, we dont support issuing 0.0.1/no defined schema after this PR